modules/sendgrid/README.md

```diff
@@ -0,0 +1,31 @@
+# Sendgrid
+
+Authenticates domain with sendgrid, assumes CNAME required are provided in variables
```

Comment:
`assumes CNAME required are provided in variables` is redundant based on the rest of the docs.